src/Repository/PlaylistSlideRepository.php

```diff
@@ -114,6 +114,73 @@ public function updateRelations(Ulid $playlistUlid, ArrayCollection $collection)
         }
     }
 
+    public function updateSlidePlaylistRelations(Ulid $slideUlid, ArrayCollection $collection)
+    {
+        $slideRepos = $this->entityManager->getRepository(Slide::class);
+        $slide = $slideRepos->findOneBy(['id' => $slideUlid]);
+        if (is_null($slide)) {
+            throw new InvalidArgumentException('Slide not found');
+        }
+
+        $playlistRepos = $this->entityManager->getRepository(Playlist::class);
+
+        $this->entityManager->getConnection()->beginTransaction();
+        try {
+            // Remove all existing relations between slides and current playlist.
+            $entities = $this->findBy(['slide' => $slide]);
+            foreach ($entities as $entity) {
+                $this->entityManager->remove($entity);
```

Comment:
This will change the order of slides in alle the playlists the slide continues to be present. They should only be remove if the do not occur in playlist anymore.